setup.py

```diff
@@ -42,7 +42,6 @@ def run_tests(self):
     license='Expat license',
     package_dir={'': 'src'},
     py_modules=['pep8ext_naming'],
-    install_requires=['flake8>=3.9.1'],
```

Comment:
We can introduce a lower limit that's higher than this, but there's no need to remove it here or rely on requirements.txt

Reply:
"It was hard for me to find what `flake8` version this tool supported", and "most people have their requirements at `requirements.txt`"

If you don't want that file, fine (it's of course your project), but "def" you don't support `flake8>=3.9.1` (at least, your tests fail to run on those versions).

The initial inspiration for this was "you don't support `flake8 >= 6.0.0`", but after doing all this work, it turned out that "something else was broken at my env". I thought you'd appreciate the work done here (hence the PR).

Reply:
No, requirements.txt isn't for libraries. Like I said, bumping the lower limit is fine. Changing radically and incorrectly how we define dependencies isn't fine. If you don't like that feedback, that's fine, you don't have to change this and we don't have to merge it. You don't need to be hostile that a project wants things done a specific way. You did extra work without talking to us and semn upset that we don't want to do it your way. That's an unreasonable expectation that we just take what you did regardless of it's quality

Reply:
> You don't need to be hostile that a project wants things done a specific way.

I am sorry that I am coming off as hostile - it was never my intention.

I merely listed out the context and timeline behind this PR - there's no (ill) emotions running here.